src/plugins/MapStitching/MapStitchCollect.vue

```diff
@@ -7,53 +7,51 @@ export default {
     settings: {},
   },
   subscriptions() {
-    async function hash(string) {
-      const hashBuffer = await crypto.subtle.digest('SHA-256', string);
-      const hashArray = Array.from(new Uint8Array(hashBuffer));
-      const hashHex = hashArray
-        .map((bytes) => bytes.toString(16).padStart(2, '0'))
-        .join('');
-      return hashHex;
-    }
+    
     var nodeDB = {};
-    const webAddress = this.settings.webAddress //this won't work inside the promises, so define a const to use.
+    const webAddress = this.settings.webAddress != undefined ?
+        this.settings.webAddress.replace(/\/$/, "") : //Remove trailing slash if present
+        undefined;
     const locationStream = GameService.getLocationStream();
+    
+    async function SendImage(Location) {
```

Comment:
You could move that to `methods` on the component and call it with `this.SendImage(location)`